src/workloads/query/multiplanner/Subplanning.yml

```diff
@@ -356,5 +356,7 @@ AutoRun:
           - standalone-sbe
           - standalone-all-feature-flags # At time of writing this will enable PM-3591.
           - standalone-classic-query-engine
+          - replica
```

Comment:
Same case - I think we could remove the non-feature-flag replica case here as there is no standard standalone case running